circle.yml

```diff
@@ -1,4 +1,30 @@
-general:
-  branches:
-    only:
-      - /circle-.*/
+machine:
+  xcode:
+    version: 7.3
+  environment:
+    XCODE_PROJECT: build/tangram.xcodeproj
+    XCODE_SCHEME: phony # Dummy value, Circle won't run without a scheme.
+checkout:
+  post:
+    - git submodule sync
+    - git submodule update --init --recursive
+test:
+  pre:
+    - set -o pipefail &&
+      mkdir build &&
+      cd build &&
+      cmake ..
+      -DPLATFORM_TARGET=ios.framework
+      -DIOS_PLATFORM=SIMULATOR
+      -DCMAKE_TOOLCHAIN_FILE=toolchains/iOS.toolchain.cmake
+      -DBUILD_IOS_FRAMEWORK=TRUE
+      -G Xcode
+  override:
+    - set -o pipefail &&
+      xcodebuild
+      -project build/tangram.xcodeproj
+      -target TangramMap
+      -configuration Release |
+      xcpretty
+  post:
+    - mv build/lib/Release/TangramMap.framework ${CIRCLE_ARTIFACTS}/
```

Comment:
If we build the universal framework then yes, we can put that among the artifacts. 

I tried running `make ios` at first and was not successful, so for my own sanity I wrote the commands explicitly. I just tried it again and it still runs into problems: https://circleci.com/gh/tangrams/tangram-es/77. If you think this can be easily fixed then I'd definitely prefer writing a single `make` command!